packages/slice-machine/server/src/api/io/CustomType.ts

```diff
@@ -13,3 +13,9 @@ export function readCustomType(path: string): CustomTypeSM {
 export function writeCustomType(path: string, customType: CustomTypeSM) {
   Files.write(path, CustomTypes.fromSM(customType));
 }
+
+export function renameCustomType(path: string, newCustomTypeName: string) {
```

Comment:
trying to think if it's worth doing run time type verification here, or not :/
if so the two functions above show how it can be done :)

Reply:
don't think I understand what do you mean

Reply:
changed